myjsonrpc.pm

```diff
@@ -53,9 +54,11 @@ sub send_json {
 
     my $wb = syswrite($to_fd, "$json");
     if (!$wb || !$wb == length($json)) {
-        confess "syswrite failed: $!" if DEBUG_JSON;
-        # remote end most likely terminated on request, silently ignore
-        return undef;
+        if (!DEBUG_JSON && $! =~ qr/Broken pipe/) {
+            bmwqemu::diag("myjsonrpc: remote end terminated connection, stopping");
+            _exit(0);
```

Comment:
Is it possible to take that change out and keep it separate (see https://github.com/os-autoinst/os-autoinst/pull/1277#discussion_r349466656).

Reply:
yes, sure. It will be handled in https://github.com/os-autoinst/os-autoinst/pull/1277#discussion_r349466656. I will mark the PR as [WIP] and I would appreciate a review of everything else included then